assignments/lambda-classes.js

```diff
@@ -1 +1,139 @@
 // CODE here for your Lambda Classes
+
+
+class Person {
+    constructor(attributes) {
+        this.name = attributes.name;
+        this.age = attributes.age;
+        this.location = attributes.location;
+        this.gender = attributes.gender;
+}
+
+speak() {
+    return `Hello my name is ${this.name} , I am from ${this.location}`
+}
+} 
+
+class Instructor extends Person {
+    constructor(attributes) {
+    super(attributes);
+    this.specialty = attributes.specialty;
+    this.favLanguage = attributes.favLanguage;
+    this.catchPhrase = attributes.catchPhrase;
+}
+demo(subject) {
+   return `Today we are learning about ${subject}`
+    
+ }   
+    
+
+
+grades(student,subject){
+   return `${student.name} receives a perfect score on ${subject}`
+}
+}
+class Student extends Instructor {
```

Comment:
go over the readme instructions again and be sure you have your inheritance chains set up properly, should Student be a subclass of Instructor?